suite/tests/api/ir_x86.c

```diff
@@ -1278,6 +1278,38 @@ test_x64_inc(void *dc)
     instr = INSTR_CREATE_inc(dc, opnd_create_reg(DR_REG_EAX));
     test_instr_encode(dc, instr, 2);
 }
+
+static void
+test_x64_vmovq(void *dc)
+{
+    /* 62 61 fd 08 d6 0c 0a vmovq  %xmm25[8byte] -> (%rdx,%rcx)[8byte] */
+    buf[0] = 0x62;
```

Comment:
How about `const byte dbuf[] = { 0x62, 0x61, 0xfd, 0x08, 0xd6, 0x0c, 0x0a };`?  Seems easier to read and more compact.

Reply:
Done.